Formula/snapcast.rb

```diff
@@ -0,0 +1,35 @@
+class Snapcast < Formula
+  desc "Synchronous multiroom audio player"
+  homepage "https://github.com/badaix/snapcast"
+  url "https://github.com/badaix/snapcast/archive/refs/tags/v0.26.0.tar.gz"
+  sha256 "166353267a5c461a3a0e7cbd05d78c4bfdaebeda078801df3b76820b54f27683"
+  license "GPL-3.0-only"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "flac"
+  depends_on "libsoxr"
+  depends_on "libvorbis"
+  depends_on "opus"
+  depends_on "pulseaudio"
+  uses_from_macos "expat"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    # Fix permissions
+    chmod(0555, "#{share}/snapserver/plug-ins/meta_mpd.py")
```

Comment:
What permissions did this have originally, and why are they wrong? Shouldn't CMake handle this?

Reply:
The original permissions were `111`. I'm not sure why they're wrong, but if nothing is changed Homebrew throws with

```
Error: Failed to read Mach-O binary: /opt/homebrew/Cellar/snapcast/0.26.0/share/snapserver/plug-ins/meta_mpd.py
```

Reply:
Can you add that to your comment? Preferably with a `FIXME`.

Reply:
Fixed in 67e1739eb40.